agent/newrelic-install.sh

```diff
@@ -221,6 +221,7 @@ arch=`(uname -m) 2> /dev/null` || arch="unknown"
 os=`(uname -s) 2> /dev/null` || os="unknown"
 
 case "${arch}" in
+  aarch64)               ;;
```

Comment:
aarch64 seems to be Linux specific. Should there also be a case for `arm64`? That's what's reported by `uname -m` on macOS running on Apple M1 chip. I'm not sure if we care for this case here, because there's macOS's logic later in the script. However that logic completely ignores `uname -m` output and assumes that macOS only runs on x86_64, which will cause problems if this script is executed on macOS.

Reply:
Since Linux is the only [supported operating system](https://docs.newrelic.com/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements), having aarch64 is enough.